react/src/hooks/useBackendAIImageMetaData.test.tsx

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable testing-library/render-result-naming-convention */
+import { useBackendAIImageMetaData } from '.';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { renderHook, waitFor } from '@testing-library/react';
+import { ReactNode } from 'react';
+
+describe('useBackendAIImageMetaData', () => {
+  const queryClient = new QueryClient();
+  const wrapper = ({ children }: { children: ReactNode }) => (
+    <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
+  );
+
+  (global as any).fetch = jest.fn(async () =>
+    Promise.resolve({
+      ok: true,
+      status: 200,
+      json: () =>
+        Promise.resolve({
+          imageInfo: {},
+          tagAlias: {
+            pytorch: 'PyTorch',
+            r: 'R',
+            'r-base': 'R',
+            py3: 'Python3',
+            ngc: 'Nvidia GPU Cloud',
+            py310: 'Python3',
+          },
+          tagReplace: {},
+        }),
+    }),
+  );
```

Comment:
I used jest.fn because suspense can't wait for the test to complete while fetching data